brightray/browser/browser_main_parts.cc

```diff
@@ -168,7 +168,9 @@ BrowserMainParts::~BrowserMainParts() {
 void OverrideAppLogsPath() {
 #if defined(OS_WIN)
   std::wstring app_name = base::UTF8ToWide(GetApplicationName());
-  std::wstring log_path = L"%HOMEDRIVE%%HOMEPATH%\\AppData\\Roaming\\";
+  std::wstring home = std::wstring(_wgetenv(L"HOMEDRIVE"));
+  std::wstring drive = std::wstring(_wgetenv(L"HOMEPATH"));
```

Comment:
I don't have much to add that hasn't already been discussed but would add one minor point: `X var = X(args)` is a strange way to instantiate something.

Instead, consider these (in order of my bias):
 * [AAA style](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/): `auto home = std::wstring{foo};`
 * uniform initializer syntax but no auto: `std::wstring home {foo};`
 * pre-C++11 style: `std::wstring home (foo);`

Reply:
Oh, also, ignorant OS_WIN question: are we _guaranteed_ that `HOMEDRIVE` and `HOMEPATH` will always return non-null? Otherwise passing this to a wstring ctor will crash us

Reply:
d'oh, I see Marshall's suggestion supercedes all of this. Yes, that solution is better :+1: